### PR TITLE
[izumi] add zksync config on iziswap dex adapter

### PR DESCRIPTION
**Pull request by LpcAries**
update iziswap dex adapter to show the volume on zksync

### CODE REVIEW

**Comment by llamatester** [comment]
The iziswap adapter exports: 
        
 
 _______________________________________
    Dexs for 13/7/2023
    _______________________________________
    
    BSC 👇
    Backfill start time: 4/4/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 9251011.961538278
    Daily volume: 38633.751985752955
    Timestamp: 1689206400
    
    
    ZKSYNC 👇
    Backfill start time: 4/4/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 621143588.0385767
    Daily volume: 7188451.9290908575
    Timestamp: 1689206400
    
    
    
    
    Done in 4.79s.

File: dexs/iziswap/index.ts [hunk 1]
```diff
@@ -54,6 +55,11 @@ const adapter: SimpleAdapter = {
       start: () => getStartTimestamp(chains[CHAIN.BSC]),
       customBackfill: customBackfill(CHAIN.BSC as Chain, fetch)
     },
+    [CHAIN.ZKSYNC]: {
```
Comment (dtmkeng) [inline dexs/iziswap/index.ts]: hi, seem chainId 324 we set is era, could you change to `ERA` ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: dexs/iziswap/index.ts (hunk 1 above)
Comment (LpcAries) [reply dexs/iziswap/index.ts]: Sorry, my miss. Code updated

**Comment by LpcAries** [comment]
I found that only the data after July 14th is displayed on the defillama page. I tested locally and encountered a bug as shown in the screenshot. Is there any way to improve this? @dtmkeng

**Pull request merged**